internal/rbd/rbd_util.go

```diff
@@ -239,9 +316,19 @@ func GetKrbdSupportedFeatures() (string, error) {
 	}
 	val, err := os.ReadFile(krbdSupportedFeaturesFile)
 	if err != nil {
-		log.ErrorLogMsg("reading file %q failed: %v", krbdSupportedFeaturesFile, err)
+		if !errors.Is(err, os.ErrNotExist) {
+			log.ErrorLogMsg("reading file %q failed: %v", krbdSupportedFeaturesFile, err)
 
-		return "", err
+			return "", err
+		}
+		attrs, err := prepareKrbdFeatureAttrs()
+		if err != nil {
+			log.ErrorLogMsg("preparing krd feature attributes failed, %v", err)
+
+			return "", err
+		}
+
+		return attrs, nil
```

Comment:
it would be cleaner to do the `strconv.FormatInt()` here, less repeated code in `prepareKrbdFeatureAttrs()`